packages/mint-components/src/components/sqm-rewards-table/columns/readme.md

```diff
@@ -0,0 +1,60 @@
+# sqm-referral-table-user-column
```

Comment:
Flagging incorrect component name

Reply:
Don't this has actually been resolved

Reply:
should be resolved now! fixed also in rewards cells